README.md

```diff
@@ -108,6 +108,14 @@ PITCHFORK_INGRESS_KAFKA_AUTO_OFFSET_RESET                      | earliest
 PITCHFORK_INGRESS_KAFKA_SESSION_TIMEOUT_MS                     | 60000
 PITCHFORK_INGRESS_KAFKA_SOURCE_TOPICS                          | zipkin
 PITCHFORK_INGRESS_KAFKA_SOURCE_FORMAT                          | JSON_V2
+PITCHFORK_INGRESS_RABBITMQ_ENABLED                             | false
+PITCHFORK_INGRESS_RABBITMQ_USER                                | guest
+PITCHFORK_INGRESS_RABBITMQ_PASSWORD                            | guest
```

Comment:
It's not safe to store passwords in clear text and pass these as properties. Can we read this pwd from a hidden file?

Reply:
I think most users will be using "guest/guest" as these brokers are running on private networks.
In this particular case I would rather use Cloud Vault to solve this, or better yet, to do nothing now and wait for feedback from users and have this discussion at a later point in time.